packages/node_modules/@webex/plugin-meetings/src/common/events/events.js

```diff
@@ -0,0 +1,27 @@
+import EventEmitter from 'events';
+
+import LoggerProxy from '../logs/logger-proxy';
+import LoggerConfig from '../logs/logger-config';
+
+/**
+ * Events
+ * plugin-meetings global
+ * Used to emit events internally between modules
+ */
+class Events extends EventEmitter {
```

Comment:
This looks nearly identical to `events-scope.js`. There should be a way to have this in one file and share between the two.

Reply:
because of inability to do multiple inheritance I stayed away from it, but I created a shared util file for them.

Reply:
Sounds good - it's a balance to not get too abstract so you can understand the code later. I find one copy/paste is fine, 3-4 should probably get wrapped up.